Repairs/app/src/main/java/com/rooio/repairs/Jobs.kt

```diff
@@ -31,7 +32,6 @@ class Jobs : NavigationBar() {
         @JvmStatic private var pendingJobs = ArrayList<JSONObject>()
         @JvmStatic private var scheduledJobs = ArrayList<JSONObject>()
         @JvmStatic private var inProgressJobs = ArrayList<JSONObject>()
-        @JvmStatic private var archivedJobs = ArrayList<JSONObject>()
```

Comment:
I'm assuming this companion object is getting fixed in either yours or Yusuf's code review?